### ISSUE TITLE
diffuser version support Ascend NPU!

### ISSUE THREAD

**Issue by designeryuan** [Tencent/HunyuanDiT#226]
i am using HunyuanDiT-v1.2-Diffusers on Ascend NPU

```
import torch
import torch_npu
from torch_npu.contrib import transfer_to_npu
from diffusers import HunyuanDiTPipeline

pipe = HunyuanDiTPipeline.from_pretrained("/model/HunyuanDiT-v1.2-Diffusers", torch_dtype=torch.float16)
pipe.to("cuda")

# You may also use English prompt as HunyuanDiT supports both English and Chinese
# prompt = "An astronaut riding a horse"
prompt = "一个宇航员在骑马"
image = pipe(prompt).images[0]
```

when i choose fp16, the pic is all black, bf16 not good, like a mask

![image](https://github.com/user-attachments/assets/8ac89191-2be7-4aa0-ad13-224620dc3d3e)

so, could u please support diffuser version for Ascend NPU, thanks!

**Comment by designeryuan** [comment]
sorry, the prompt is "一只小猫抓蝴蝶，阳光灿烂"
